docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Kaldi.md

```diff
@@ -50,7 +50,7 @@ The following list provides the Kaldi\*-specific parameters.
 
 ```sh
 Kaldi-specific parameters:
-  --counts COUNTS       A file name with full path to the counts file
+  --counts COUNTS       A file name with full path to the counts file or empty string id counts are included in model file
```

Comment:
Not clear what value is specified for `--count` if count values are inside the model. Could you please elaborate it in the response to my comment.